data/mods/Magiclysm/items/spellbooks.json

```
@@ -172,7 +181,71 @@
     "color": "light_gray",
     "use_action": {
       "type": "learn_spell",
-      "spells": [ "jolt", "windstrike", "windrun", "storm_hammer", "lightning_bolt", "lightning_blast" ]
+      "spells": [
+        "electric_arts",
+        "jolt",
+        "windstrike",
+        "windrun",
+        "storm_hammer",
+        "lightning_bolt",
+        "lightning_blast",
+        "shocking_dash",
+        "toxtricity"
+      ]
+    }
+  },
+  {
+    "id": "bioelectricity",
+    "type": "BOOK",
+    "category": "manuals",
+    "//": "6 Stormshaper spells",
+    "name": { "str": "Bioelectricity, An Arcane Approach", "str_pl": "copies of Bioelectricity, An Arcane Approach" },
+    "description": "Small, but thick book, that covers most recent development in the field, including real examples.",
```

```suggestion
    "description": "Small, but thick book, that covers the most recent developments in the field, including real examples.",
```
This sounds like a generic academic tome.

---

Because i copypasted it from the real book of almost the same name